controls/nist_ocp4.yml

```diff
@@ -1092,14 +1092,14 @@ controls:
 - id: AC-6(8)
   status: supported
   notes: |-
-    Red Hat OpenShift provides security context constraints (SCC) [1] that control permissions for actions that a pod can perform and what resources a pod can access. 
+    Red Hat OpenShift provides security context constraints (SCC) [1] that control permissions for actions that a pod can perform and what resources a pod can access.
```

Comment:
Fixing all these random whitespace issues in https://github.com/ComplianceAsCode/content/pull/13261